src/link_manager/LinkManagerStateMachine.cpp

```diff
@@ -902,10 +904,10 @@ void LinkManagerStateMachine::handleDefaultRouteStateNotification(const std::str
 //
 void LinkManagerStateMachine::shutdownOrRestartLinkProberOnDefaultRoute()
 {
-    MUXLOGWARNING(boost::format("%s: default route state: %s") % mMuxPortConfig.getPortName() % mDefaultRouteState);
+    MUXLOGDEBUG(mMuxPortConfig.getPortName());
```

Comment:
What can you make out when you see this debug log?

Reply:
> What can you make out when you see this debug log?

I used this log to confirm that there is check to shutdown or restart link prober when I updated mux mode or default route.